sources/packages/backend/apps/api/src/services/student-file/student-file.service.ts

```diff
@@ -113,17 +112,15 @@ export class StudentFileService extends RecordDataModelService<StudentFile> {
    * @param uniqueFileNames list of unique file names.
    * @param fileOrigin origin of the file being saved.
    * @param groupName group name of the file being save.
-   * @param sendNotification optional notification message to be sent.
-   * @param metadata optional metadata of the file being save.
+   * @param options file upload options.
```

Comment:
When we move the option to an object I would recommend adding the comments there.
If we create the inline object we should also add the comments here. Either way, the comments are welcome.

Reply:
did you add it? I am not seeing it

Reply:
Added to the properties in options object.
![image](https://user-images.githubusercontent.com/54600590/206045859-5131fdcb-7962-4101-80e4-3b3aea6c1beb.png)